policies/cla.html

```diff
@@ -34,9 +34,19 @@
           </p>
 
           <p>
-          A submission is considered trivial if the author and all reviewers
-          agree that it is. In this case a CLA is not required and the
-          commit text should include "CLA: trivial" on a line by itself.
+	  A submission is trivial if it is considered trivial under copyright
+	  law. Since we are not lawyers, we place the bar for trivial
+	  contributions very high. Corrections of grammatical or
+	  typographical errors (including misspelled function names in manual
+	  pages), simple whitespace changes and in some cases one-line
+	  bugfixes might be accepted as trivial without requiring a CLA.
+          </p>
+
+          <p>
+	  In practice it is required that the author (in the git commit
```

Comment:
Add comma after "In practice"?  (If yes, it's my fault.)

Reply:
Done.